pilot/pkg/bootstrap/server.go

```diff
@@ -565,14 +564,17 @@ func (s *Server) initIstiodAdminServer(args *PilotArgs, wh *inject.Webhook) erro
 		Addr:    args.ServerOptions.HTTPAddr,
 		Handler: s.httpMux,
 	}
-
+	securedDebug := true
+	if s.httpServer.Addr != "localhost:8080" {
```

Comment:
maybe we want to use https://golang.org/pkg/net/#IP.IsLoopback instead. They may change the port or format a bit

Reply:
fixed